streaming/src/main/scala/org/apache/spark/streaming/receiver/ActorReceiver.scala

```diff
@@ -69,13 +68,60 @@ object ActorSupervisorStrategy {
  *       should be same.
  */
 @DeveloperApi
-trait ActorHelper extends Logging{
+abstract class ActorReceiver extends Actor {
 
-  self: Actor => // to ensure that this can be added to Actor classes only
+  /** Store an iterator of received data as a data block into Spark's memory. */
+  def store[T](iter: Iterator[T]) {
```

Comment:
I duplicated these `store` methods in `ActorReceiver` and `JavaActorReceiver` because I don't want to expose `ActorHelper` as a public API. Note: if we put `store` methods in `ActorHelper`, then it must be a public API so as to appear in the Scala/Java docs.